DfE.FindInformationAcademiesTrusts/Services/ExportService.cs

```diff
@@ -0,0 +1,99 @@
+﻿using ClosedXML.Excel;
+using DfE.FindInformationAcademiesTrusts.Data;
+
+namespace DfE.FindInformationAcademiesTrusts.Services.Export
+{
+    public interface IExportService
+    {
+        byte[] ExportAcademiesToSpreadsheetUsingProvider(Data.Trust? trust, Trust.TrustSummaryServiceModel? trustSummary);
+    }
+
+    public class ExportService : IExportService
+    {
+
+        public byte[] ExportAcademiesToSpreadsheetUsingProvider(Data.Trust? trust, Trust.TrustSummaryServiceModel? trustSummary)
+        {
+            var headers = new List<string>
```

Comment:
I think I prefer the headers and data extractor as static readonly fields rather than part of the method.
Totally personal preference though so feel free to keep it like this if you prefer it.